env/.env.dev.user

```diff
@@ -0,0 +1,8 @@
+# This file includes environment variables that will not be committed to git by default. You can set these environment variables in your CI/CD system for your project.
+
+# If you're adding a secret value, add SECRET_ prefix to the name so Teams Toolkit can handle them properly
+# Secrets. Keys prefixed with `SECRET_` will be masked in Teams Toolkit logs.
+SECRET_BOT_PASSWORD=
+SECRET_AZURE_OPENAI_KEY=<your key>
+SECRET_AZURE_OPENAI_MODEL_DEPLOYMENT_NAME=<your model name>
```

Comment:
minor: for non secret envs, we could put to `.env.dev`. Can see how other AI templates organizes the envs and align with them.

Reply:
considering